examples/arc.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+#
+# Test for a push-button monome arc
+
+import asyncio
+import monome
+
+class ExampleArc(monome.Arc):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+class ExampleArcApp(monome.ArcApp):   
+    def __init__(self):
+        arc = ExampleArc()
+        self.pos = [0, 0, 0, 0];
+        super().__init__(arc)
+        
+    def on_arc_ready(self):
+        print('Ready, clearing all rings...')
+        for n in range(0,4):
+            self.arc.ring_all(n,0)
```

Comment:
please check that the example code follows [PEP-8](https://www.python.org/dev/peps/pep-0008/), e.g. commas should be followed with a space.

Reply:
Thanks, yes sorry. I do not regularly code in python so these slipped through. Plus I did this originally in notepad++!
I have linted it with VisualStudioCode now but I'm not sure if it used PEP-8.

Reply:
Done now